core/capabilities/targets/write_target.go

```diff
@@ -63,37 +37,42 @@ func NewEvmWrite(chain legacyevm.Chain, lggr logger.Logger) *EvmWrite {
 		nil,
 	)
 
-	return &EvmWrite{
-		chain,
+	logger := lggr.Named("WriteTarget")
+
+	return &WriteTarget{
+		cr,
+		cw,
+		forwarderAddress,
 		info,
-		lggr.Named("EvmWrite"),
+		logger,
 	}
 }
 
 type EvmConfig struct {
-	ChainID uint
 	Address string
 }
 
-// TODO: enforce required key presence
-
-func parseConfig(rawConfig *values.Map) (EvmConfig, error) {
-	var config EvmConfig
-	configAny, err := rawConfig.Unwrap()
-	if err != nil {
+func parseConfig(rawConfig *values.Map) (config EvmConfig, err error) {
+	if err := rawConfig.UnwrapTo(&config); err != nil {
 		return config, err
 	}
-	err = mapstructure.Decode(configAny, &config)
-	return config, err
+	if !common.IsHexAddress(config.Address) {
+		return config, fmt.Errorf("'%v' is not a valid address", config.Address)
+	}
+	return config, nil
 }
 
-func (cap *EvmWrite) Execute(ctx context.Context, request capabilities.CapabilityRequest) (<-chan capabilities.CapabilityResponse, error) {
-	cap.lggr.Debugw("Execute", "request", request)
-	// TODO: idempotency
-
-	txm := cap.chain.TxManager()
+func success() <-chan capabilities.CapabilityResponse {
+	callback := make(chan capabilities.CapabilityResponse)
+	go func() {
+		callback <- capabilities.CapabilityResponse{}
```

Comment:
Don't we want to put some info in the response? For example whether we attempted transmission or not. Maybe not important atm but let's consider it in the future.